app/views/claims/support/claims/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= render "claims/support/primary_navigation", current: :claims %>
-<% content_for(:page_title) { t(".heading") } %>
+<% content_for(:page_title) { t(".page_title") } %>
```

Comment:
You probably need to pass a variable in here 
![Screenshot from 2024-04-18 14-31-29](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/fba3fa6b-4c10-49d6-bce0-bc7cde952124)